contracts/deploy/base/009_upgrade_vault.js

```diff
@@ -0,0 +1,77 @@
+const { deployOnBaseWithGuardian } = require("../../utils/deploy-l2");
+const { deployWithConfirmation } = require("../../utils/deploy");
+const addresses = require("../../utils/addresses");
+const { utils } = require("ethers");
+
+module.exports = deployOnBaseWithGuardian(
+  {
+    deployName: "009_upgrade_vault",
+  },
+  async ({ ethers }) => {
+    const cOETHbVaultProxy = await ethers.getContract("OETHBaseVaultProxy");
+    const cOETHbVault = await ethers.getContractAt(
+      "IVault",
+      cOETHbVaultProxy.address
+    );
+    const cOETHbDripperProxy = await ethers.getContract("OETHBaseDripperProxy");
+
+    // Deploy new implementation
+    const dOETHbVaultCore = await deployWithConfirmation("OETHBaseVaultCore", [
+      addresses.base.WETH,
+    ]);
+    const dOETHbVaultAdmin = await deployWithConfirmation("OETHBaseVaultAdmin");
+
+    return {
+      actions: [
+        {
+          // 1. Upgrade VaultCore
+          contract: cOETHbVaultProxy,
```

Comment:
to upgrade the vault code we only need to execute this function. Why are other governance configuration functions present?

Reply:
I actually haven't configured those other params on the Vault during deploy since we haven't made any decision on those. We need these configured before going live